### ISSUE TITLE
NVDA is not reading List item's and its decendants  aria-label information when navigating items using Insert+right/insert+left arrows

### ISSUE THREAD

**Issue by RajeswariAddepalli23** [nvaccess/nvda#9069]
### Steps to reproduce:

1. Switch on the NVDA
2. Bring focus to input text control.
3. Navigate to the controls using Insert + Right/Insert + Left arrows to make back and forth traversal to the controls.
     
### Actual behavior:
1. Focus is not getting to the list items and their descends.
2. Not reading the aria-label information them.  

### Expected behavior:
1. Focus should traverse to all the elements like Forms, Pages and the  buttons.
2. Should be able to read aria-label information of them.

Note : For reference, the same sample is working fine with Narrator screen reader.    

Sample provided as below:
https://jsfiddle.net/8us2y941/

#### NVDA version:
2018.2.1
#### Windows version:
Win 10
#### Does the issue still occur after restarting your PC?
No
#### Have you tried any other versions of NVDA?
No

**Comment by DrSooom** [comment]
@RajeswariAddepalli23: Please list all browsers (incl. version) in which this issue occurs. Thanks.

**Comment by Adriani90** [comment]
@RajeswariAddepalli23 I assume you tried to use object navigation here and expected to get the labels "pages" and "forms" and their child elements "remove pages" and "remove forms". In NVDA you need to walk by pressing insert+shift+right arrow, not insert+right arrow. This works with keyboard layout set to laptop in NVDA's settings. Of course you can also use the numeric block on a computer keyboard for this without having to hold down the insert key. For the numeric block to work with this you have to turn off the numlock and you need to set the keyboard layout to desktop in NVDA's settings.
This works as expected in NVDA. Closing as works for me.
But feel free to comment on this if I missunderstood something here and we can reopen. Thanks.

**Comment by cppqtdev** [comment]
Hi Team 
I am facing one issue with NVDA Screen Reader when the focus goes to the Listview in Qt /QML Application  
it is not reading 1 of 5 ,Let's total 5 item in the list and current item is 1 . then it should read name of the item and then read selected and then 1 of 5 or 1 of 5 item selected .


Note : When I tested this on Window Inbuilt Screen reader like narrator it read properly and also JAWS also read same behavior as Narrator screen reader .


Please support if possible .What we need to do ?

**Comment by seanbudd** [comment]
@cppqtdev please open a new issue. We cannot track, triage or address comments on old, closed issues.
